cmake/Modules/PkgBuildConfig.cmake

```diff
@@ -1,6 +1,8 @@
 # pkg-config file generation
 #
 
+INCLUDE(GNUInstallDirs)
```

Comment:
I suppose this doesn't hurt but if you swapped the includes in `src/`, this inherit shouldn't be necessary. I think the general idea is to include `GNUInstallDirs` in top CMakeLists since it's a typical project-global module.